src/translations/screens/Dashboard.ts

```diff
@@ -8,6 +8,16 @@ const DashboardTexts = {
     },
   },
   buyButton: _('Kjøp billetter', 'Buy tickets', 'Kjøp billettar'),
+  announcemens: {
+    header: _('Aktuelt', 'Announcements', 'Aktuelt'),
+    button: {
+      accessibility: _(
+        'Trykk for å lese mer',
+        'Press to read more',
+        'Trykk for å lese meir',
+      ),
```

Comment:
Hints like these should say "activate to …" instead of "press to …". Accessibility users may for example use voice activation instead of pressing to activate.

Reply:
Ok, it's not consistent throughout the app. What would that be in Norwegian? "Aktiver ..."?

Reply:
Yeah, "Aktiver for"/"Activate to" is the correct phrasing. I made a quick search in the app now and think we seem fairly consistent, I just found a few recently added Nynorsk-texts with the wrong phrasing.